src/js/tabs/receiver.js

```diff
@@ -336,8 +363,39 @@ receiver.initialize = function (callback) {
                     },
                 });
             }
-        }
 
+            console.log(FC.RX_CONFIG.rxSpiProtocol);
+
+            if (FC.FEATURE_CONFIG.features.isEnabled('RX_SPI') &&
```

Comment:
this complete block needs 1 intend tab

Reply:
Thanks for the review. Should be fixed in last push.